types/mathjs/index.d.ts

```diff
@@ -1,22 +1,22 @@
-// Type definitions for mathjs
+// Type definitions for mathjs 3.20
 // Project: http://mathjs.org/
-// Definitions by: Ilya Shestakov <https://github.com/siavol>
+// Definitions by: Ilya Shestakov <https://github.com/siavol>,
+// 				   Andy Patterson <https://github.com/andnp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
```

Comment:
I'm using `Record` in here, which raises the minimum TS version to 2.1. If there are any objections to this, I can roll that back, but I want to be able to use more up-to-date TS as I continue working on these types.